.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,9 @@
 <!--  Thanks for sending a pull request!  Here are some tips for you:
 1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+2. Ensure you have ran the appropriate tests for your PR: https://github.com/kubernetes/community/blob/master/contributors/devel/testing.md
```

Comment:
i think you need to mention adding tests, like ut and e2e .

Reply:
> i think you need to mention adding tests, like ut and e2e as part of their PR.

How about "Ensure you have added or ran the appropriate tests for your PR"?

Reply:
Modified as @nikhita  said.